plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.scss

```diff
@@ -157,6 +161,25 @@ $font-sf-pro-display: "SF Pro Text", -apple-system, BlinkMacSystemFont, "Segoe U
 			flex-direction: column;
 			gap: 16px;
 
+			&__title {
+				display: flex;
+				gap: 4px;
+				align-items: center;
+
+				&__h3 {
+					text-transform: uppercase;
+					font-size: 12px;
+					font-weight: 300;
```

Comment:
I found the combination of uppercase with a pretty thin font weight a little hard to read. It looks like font weight is 500 in the designs - WDYT about using that instead? 🙂

Reply:
Also looks like font color should be `$gray-900`

Reply:
Oh indeed, thanks for flagging it. Fixed in fe74cb1fa70b8cc6df4d08a90e70032055a91061